Automattic/jetpack
# Lazy-images: Move JS build into the package itself

#### Changes proposed in this Pull Request:

Now that we can run `yarn build` during the mirroring, we can move the
build into the package itself rather than doing it at the monorepo
level.

We can also remove the minified JS from the monorepo, since we can now
set the "production-include" attribute for those files. We should still
switch over to calypso-build at some point, but this will at least make
it cleaner until then.

#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `.github/files/update-project-mirrors.sh` to check out the mirror repo into /tmp and generate the commit that would be pushed.
* Do `git archive` in that local copy of the mirror repo.
* Check that the contents of the archive are similar to what you currently get from packagist (e.g. in an empty directory, execute `composer require --prefer-dist automattic/jetpack-lazy-images=@dev`), no missing or added files.

#### Proposed changelog entry for your changes:


* None needed.
